.github/utils/tutorials.sh

```diff
@@ -73,15 +83,38 @@ for script in $scripts_to_run; do
     fi
 
     if [[ "$script" == *".py" ]]; then
-        time python $script
+        output=$(time python $script)
     else
-        sudo $python_path/bin/ipython -c "%run $script"
+        output=$(sudo $python_path/bin/ipython -c "%run $script")
     fi
 
-    if [ ! $? -eq 0 ]; then
+    echo $output > $script-output.txt
+    if [ $? -eq 0 ]; then
+        echo "Execution completed successfully. See artifacts for the tutorial's output."
+    else
+        echo "===================================================="
+        echo "|  $script FAILED!"
+        echo "===================================================="
+        echo "Output of the execution: "
+        echo $output
         failed=$failed" "$script
     fi
 
+    # Restart the containers
+    if [[ "$make_python_path_editable" == "RESTART" ]]; then
+        docker stop elasticsearch
+        docker stop tika
+        docker stop graphdb
+
+        docker rm elasticsearch
+        docker rm tika
+        docker rm graphdb
+        
+        docker run -d -p 9200:9200 --name elasticsearch -e "discovery.type=single-node" -e "ES_JAVA_OPTS=-Xms128m -Xmx256m" elasticsearch:7.9.2
```

Comment:
If we use the same container name here, it will be an elasticsearch instance that was previously used and thus one that will still contain the data from previous tutorials, right? I think we would like to prevent that from happening.